### PR TITLE
Added double pattern to fix bug  #1323

### PR DESCRIPTION
**Pull request by mchav**
@cvogt unclear on how exactly to test but were you looking for something of this nature?

### CODE REVIEW

**Comment by Sumith1896** [comment]
We'll need tests too.

**Comment by hvesalai** [comment]
This PR will not work. I'll add a comment to point out the bug.

File: slick/src/main/scala/slick/jdbc/PostgresProfile.scala [hunk 1]
```diff
@@ -65,12 +65,14 @@ trait PostgresProfile extends JdbcProfile {
     override def createColumnBuilder(tableBuilder: TableBuilder, meta: MColumn): ColumnBuilder = new ColumnBuilder(tableBuilder, meta) {
       val VarCharPattern = "^'(.*)'::character varying$".r
       val IntPattern = "^\\((-?[0-9]*)\\)$".r
+      val DoublePattern = "[-+]?[0-9]*\\.?[0-9]+([eE][-+]?[0-9]+)?".r
```
Comment (hvesalai) [inline slick/src/main/scala/slick/jdbc/PostgresProfile.scala]: Unlike in the IntPattern, which captures the sign and all the digits, here you capture only the exponential part `([eE][-+]?[0-9]+)`

File: slick/src/main/scala/slick/jdbc/PostgresProfile.scala [hunk 2]
```diff
@@ -65,12 +65,14 @@ trait PostgresProfile extends JdbcProfile {
...
       override def default = meta.columnDef.map((_,tpe)).collect{
         case ("true","Boolean")  => Some(Some(true))
         case ("false","Boolean") => Some(Some(false))
         case (VarCharPattern(str),"String") => Some(Some(str))
         case (IntPattern(v),"Int") => Some(Some(v.toInt))
         case (IntPattern(v),"Long") => Some(Some(v.toLong))
+        case (DoublePattern(v), "Double") => Some(Some(v.toDouble))
```
Comment (hvesalai) [inline slick/src/main/scala/slick/jdbc/PostgresProfile.scala]: Yet hear you use the DoublePattern's unapply to fetch get input matching to the capture group. This means you are calling `toDouble` on a string that only consists of the exponentialPart (e.g. "e-23")

**Comment by hvesalai** [comment]
@mchav would you care to attempt improving the PR and adding a test case to show that it works.

**Comment by cvogt** [comment]
maybe @trevorsibanda can take a looks? @mchav is working for Google now and I don't think he still dabbles in Scala land and I am not sure what kind of authorization he has to work on OSS projects outside Google.

**Comment by hvesalai** [comment]
@trevorsibanda cvogt suggested you could take a look at this

**Comment by SethTisue** [comment]
closing for inactivity

**Pull request closed without merging**